src/ui/components/uicontroller/UiContainer.tsx

```diff
@@ -113,6 +135,20 @@ export const BOTTOM_UI_CONTAINER_STYLE: ViewStyle = {
   paddingRight: 10,
 };
 
+/**
+ * The default style for the ad container.
+ */
+export const AD_UI_CONTAINER_STYLE: ViewStyle = {
+  position: 'absolute',
+  left: 0,
+  right: 0,
+  bottom: 0,
+  zIndex: 1,
+  paddingBottom: 10,
+  paddingLeft: 10,
+  paddingRight: 10,
+};
```

Comment:
This seems identical to `BOTTOM_UI_CONTAINER_STYLE`?

Could also do:
```suggestion
export const AD_UI_CONTAINER_STYLE: ViewStyle = BOTTOM_UI_CONTAINER_STYLE;
```